src/module_2/module_2_groceries.ipynb

```diff
@@ -0,0 +1,3261 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# MODULE 2: GROCERIES E-COMMERCE"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import seaborn as sns\n",
+    "\n",
+    "plt.style.use(\"ggplot\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### PULLING THE DATASETS"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_path = \"/home/unai/datasets/groceries/\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<class 'pandas.core.frame.DataFrame'>\n",
+      "Index: 8773 entries, 10 to 64538\n",
+      "Data columns (total 6 columns):\n",
+      " #   Column          Non-Null Count  Dtype         \n",
+      "---  ------          --------------  -----         \n",
+      " 0   id              8773 non-null   int64         \n",
+      " 1   user_id         8773 non-null   object        \n",
+      " 2   created_at      8773 non-null   datetime64[us]\n",
+      " 3   order_date      8773 non-null   datetime64[us]\n",
+      " 4   user_order_seq  8773 non-null   int64         \n",
+      " 5   ordered_items   8773 non-null   object        \n",
+      "dtypes: datetime64[us](2), int64(2), object(2)\n",
+      "memory usage: 479.8+ KB\n"
+     ]
+    }
+   ],
+   "source": [
+    "orders = pd.read_parquet(f\"{data_path}/orders.parquet\")\n",
```

Comment:
Take a look at [Pathlib](https://docs.python.org/3/library/pathlib.html) for managing paths, since using strings are error prone (e.g. trailing `/` when concatenating)

Reply:
Great